src/planner_test.cc

```diff
@@ -831,6 +831,40 @@ TEST(PlannerTest, DetectOutRangeAcceleration) {
   VerifyCommonExpectations(plantest.segments(), *plantest.GetConfig());
 }
 
+// If we start from zero speed and enqueue a line of segments
+// at constat max feedrate we expect the profile to always be symmetrical.
+TEST(PlannerTest, StraightLine_ProfileSymmetry) {
+  MachineControlConfig *config = new MachineControlConfig();
+  InitTestConfig(config);
+  // The acceleration should stay high enough to avoid making
+  // to avoid numerical issues due to square root calculations
+  // for which the acceleration and deceleration, if they
+  // require too many steps, will end up being very inaccurate.
+  PlannerHarness plantest(0, 0, config);
+  unsigned kNumSteps = 200;
+  AxesRegister pos = {};
+
+  for (unsigned i = 0; i < kNumSteps; ++i) {
+    pos[AXIS_X] += 10;
```

Comment:
If we have a constant `kStepsPerSegment`, this number doesn't come out of thin air (and it would allow to change it for experiments)

Reply:
(solved)